lpd/tests/test_models.py

```diff
@@ -1,11 +1,317 @@
+        for answer_option in [answer_option0, answer_option1, answer_option2]:
+            self.assertIn(answer_option, answer_options)
+
+        question.randomize_options = False
+        question.save()
+        answer_options = list(question.get_answer_options())
+        # Question is configured *not* to display answer options in random order,
+        # so we need to check if `get_answer_options` returns answer options in alphabetical order.
+        self.assertEqual(len(answer_options), 3)
+        self.assertEqual(answer_options, [answer_option1, answer_option2, answer_option0])
+
+
+class MultipleChoiceQuestionTests(TestCase, QuantitativeQuestionTestMixin):
+    """MultipleChoiceQuestion model tests."""
+
+    def setUp(self):
+        self.question_factory = MultipleChoiceQuestionFactory
+
+    def test_str(self):
+        """
+        Test string representation of `MultipleChoiceQuestion` model.
+        """
+        question = MultipleChoiceQuestionFactory(question_text='Is this a multiple choice question?')
+        self.assertEqual(str(question), 'MultipleChoiceQuestion 1: Is this a multiple choice question?')
+
+    def test_type(self):
+        """
+        Test that `type` property returns appropriate value.
+        """
+        mcq = MultipleChoiceQuestionFactory(max_options_to_select=1)
+        mrq = MultipleChoiceQuestionFactory(max_options_to_select=5)
+        self.assertEqual(mcq.type, QuestionTypes.MCQ)
+        self.assertEqual(mrq.type, QuestionTypes.MRQ)
+
+
+class RankingQuestionTests(TestCase, QuantitativeQuestionTestMixin):
+    """RankingQuestion model tests."""
+
+    def setUp(self):
+        self.question_factory = RankingQuestionFactory
+
+    def test_str(self):
+        """
+        Test string representation of `RankingQuestion` model.
+        """
+        question = RankingQuestionFactory(question_text='Is this a ranking question?')
+        self.assertEqual(str(question), 'RankingQuestion 1: Is this a ranking question?')
+
+    def test_type(self):
+        """
+        Test that `type` property returns appropriate value.
+        """
+        question = RankingQuestionFactory()
+        self.assertEqual(question.type, QuestionTypes.RANKING)
+
+    def test_unranked_option_value(self):
+        """
+        Test that `unranked_option_value` returns appropriate value.
+        """
+        for rank in [3, 5, 10]:
+            RankingQuestionFactory(number_of_options_to_rank=rank)
+        self.assertEqual(RankingQuestion.unranked_option_value(), 11)
+
+
+class LikertScaleQuestionTests(TestCase, QuantitativeQuestionTestMixin):
+    """LikertScaleQuestion model tests."""
+
+    def setUp(self):
+        self.question_factory = LikertScaleQuestionFactory
+
+    def test_str(self):
+        """
+        Test string representation of `LikertScaleQuestion` model.
+        """
+        question = LikertScaleQuestionFactory(question_text='Is this a Likert scale question?')
```

Comment:
I guess this is a nit but if we do:
```python
self.question_factory = LikertScaleQuestionFactory
```
 in the `setUp` method, then we could do here:
```python
question = self.question_factory(question_text='Is this a Likert scale question?')
```

(It's the same thing for the other test classes, I think)